content/ko/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,158 @@
+---
+
+
+title: Topology Aware Hints
+content_type: concept
+weight: 45
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+_토폴로지 인지 힌트_ 는 클라이언트가 엔드포인트를 어떻게 사용해야 하는지에 대한 제안을 포함시킴으로써 
+토폴로지 인지 라우팅을 가능하게 한다. 
+이러한 접근은 엔드포인트슬라이스(EndpointSlice) 및 엔드포인트(Endpoint) 오브젝트의 소비자(consumer)가 이용할 수 있는 메타데이터를 추가하며, 
+이를 통해 해당 네트워크 엔드포인트로의 트래픽이 근원지까지 더 가깝게 라우트될 수 있다.
+
+예를 들어, 비용을 줄이거나 또는 네트워크 성능을 증대시키기 위해, 
+인접성을 고려하여 트래픽을 라우트할 수 있다.
+
+<!-- body -->
+
+## 동기
```

Comment:
Most readers would interpret like synchronous, rather than motivation by just seeing "동기" word - would you change the word or also add English with parenthesis?